### PR TITLE
Feature/table

### PR DESCRIPTION
**Pull request by e2o**

### CODE REVIEW

**Comment by bertyhell** [comment]
```
./src/components/Table/Table.stories.tsxModule Warning (from ./node_modules/eslint-loader/index.js):  Line 1:17:  'cloneElement' is defined but never used                                                                                                                                                                                     @typescript-eslint/no-unused-vars  Line 54:5:  React Hook useEffect has missing dependencies: 'data' and 'sort'. Either include them or remove the dependency array. You can also do a functional update 'setData(d => ...)' if you only need 'data' in the 'setData' call  react-hooks/exhaustive-deps
```

File: src/components/Table/Table.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+				columns={COLUMNS}
+				data={DATA}
+				rowKey="id"
+				renderCell={(row, cell) => (
+					<Fragment>
+						{(function() {
+							switch (cell) {
+								case 'dog':
+								case 'cat':
+									return row[cell] ? 'Yes' : 'No';
```
Comment (bertyhell) [inline src/components/Table/Table.stories.tsx]: would have been cool to use a checkmark icon and a cross icon :)

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+			className={classNames('c-table', {
+				'c-table--styled': styled || bordered,
+				'c-table--bordered': bordered,
+			})}
+		>
+			{columns.length > 0 && (
+				<thead>
+					<tr>
+						{columns.map(heading => (
+							<th key={heading.id} onClick={() => heading.sortable && onColumnClick(heading.id)}>
```
[the remark this answers is not in the document]
Comment (e2o) [reply src/components/Table/Table.tsx]: Could do that, would needed to be included in the styles though. I'll add it in JIRA for when we have access to editing the styling.

File: src/components/Table/Table.stories.tsx (hunk 1 above)
Comment (e2o) [reply src/components/Table/Table.stories.tsx]: I'll add it to the story to show the possibilities indeed.

**Pull request merged**